docs/examples/gallery_streaming.py

```diff
@@ -119,9 +119,10 @@ def answer(self, messages):
         "/chats",
         json={
             "name": "Tutorial REST API",
-            "documents": [document],
+            "input": [document],
             "source_storage": source_storages.RagnaDemoSourceStorage.display_name(),
             "assistant": DemoStreamingAssistant.display_name(),
+            "corpus_name": None,
```

Comment:
We probably want to have this as default value as well.

Reply:
Is this still valid after your comments in #493?